recipes/sentencepiece/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "sentencepiece" %}
+{% set version = "0.1.85" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/google/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: dd4956287a1b6af3cbdbbd499b7227a859a4e3f41c9882de5e6bdd929e219ae6
+
+build:
+  number: 0
+
```

Comment:
Add `skip: True  # [py<3.6]` to the `build` section here, and remove `>=3.6` from python in the `host` and `run` sections below. Non noarch recipes do not support python and r-base selectors in the `requirements` section.